multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -530,7 +529,7 @@ def fit(
         column_types: Optional[dict] = None,
         holdout_frac: Optional[float] = None,
         teacher_predictor: Union[str, MultiModalPredictor] = None,
-        seed: Optional[int] = 123,
+        seed: Optional[int] = 0,
```

Comment:
We may need to compare the performance on AutoMM benchmark datasets before and after changing the seed. If there is no performance drop, we can make this change.

Reply:
If we have a performance drop, that just means we were overfit on the seed. It shouldn't be a blocker for merging.

Reply:
But yes, I would recommend a benchmark run on this PR, since there are other impactful changes, such as using stratification during splits.

Reply:
@tonyhoo Do we need to test this PR on the vision and text benchmarks?

Reply:
Can we use the tutorial generated from this PR as an indicator on the performance impact? If there are some concerns raised from there, I am fine rerun the NLP and CV benchmark